crates/store/re_chunk_store/src/gc.rs

```diff
@@ -352,7 +353,35 @@ impl ChunkStore {
                     !chunk_ids.is_empty()
                 });
 
-                for temporal_chunk_ids_per_component in temporal_chunk_ids_per_entity.values_mut() {
+                // Component-less indices
+                for temporal_chunk_ids_per_timeline in temporal_chunk_ids_per_entity.values_mut() {
+                    for temporal_chunk_ids_per_time in temporal_chunk_ids_per_timeline.values_mut()
+                    {
+                        let ChunkIdSetPerTime {
+                            max_interval_length: _,
+                            per_start_time,
+                            per_end_time,
+                        } = temporal_chunk_ids_per_time;
```

Comment:
What is `ChunkIdSetPerTime::per_start_time` and `per_end_time`? They lack docstrings

Reply:
Added docs